packages/experimental/Drawer/src/DrawerTokens.ts

```diff
@@ -0,0 +1,40 @@
+import type { Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { DrawerTokens } from './Drawer.types';
+
+export const defaultDrawerTokens: TokenSettings<DrawerTokens, Theme> = (t: Theme) =>
+  ({
+    backdropColor: '#00000080', // to be replace with token in future based on design guidance
+    drawerBackgroundColor: t.colors.neutralBackground2,
+    handleBackgroundColor: t.colors.neutralStroke1,
+    handleWidth: globalTokens.size360,
+    handleHeight: globalTokens.size40,
+    handleCornerRadius: globalTokens.size20,
+    handleAlignment: 'center',
+    handleMarginTop: globalTokens.size80,
+    handleMarginBottom: globalTokens.sizeNone,
+    handleMarginEnd: globalTokens.sizeNone,
+    handleMarginStart: globalTokens.sizeNone,
+    drawerElevation: globalTokens.size100,
+    drawerCornerRadius: globalTokens.corner.radius120,
+    top: 0,
```

Comment:
I don't see this token defined in your Drawer.types so this probably doesn't do anything. Also, setting `left` `right` and `bottom` to numeric values in the conditions below isn't doing anything either.

If you want to set these as tokens on the Drawer, you want to define them in types first (possibly rethink naming conventions since it's confusing with the position values). Then also in Drawer.styling, you want to map each token to the corresponding style prop for each slot

Reply:
I have tokens for each position in the types and reference them in styling files worked out.

Reply:
leftPosition?: number;
  rightPosition?: number;
  bottomPosition?: number;
  topPosition?: number;
  
  
  Added this to take the number coordinates of according to the position.